### PR TITLE
Remove redundant type conversion

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

File: cmd/chartmuseum/main.go [hunk 1]
```diff
@@ -22,11 +22,11 @@ import (
 	"os"
 	"strings"
 
-	"github.com/chartmuseum/storage"
 	"helm.sh/chartmuseum/pkg/cache"
 	"helm.sh/chartmuseum/pkg/chartmuseum"
 	"helm.sh/chartmuseum/pkg/config"
 
+	"github.com/chartmuseum/storage"
```
Comment (jdolitsky) [inline cmd/chartmuseum/main.go:29]: Can we put this import back w/ the other chartmuseum ones?

Comment (hezhizhen) [reply cmd/chartmuseum/main.go:29]: Do you mean mixing imports from `helm.sh/chartmuseum` and `github.com/chartmuseum`?

**Comment by hezhizhen** [comment]
@jdolitsky How do you group imports? I didn't find anything about it.

**Comment by jdolitsky** [comment]
@hezhizhen - actually you're right, should be how you had it.. Dont have a documented process, just trying to follow same process in all files. I will go ahead and merge though, its not a big deal.

**Pull request merged**